ops/docker/dockerfile/Dockerfile.gpu

```diff
@@ -0,0 +1,54 @@
+ARG CUDA_VERSION_ARG=notset
+FROM nvidia/cuda:$CUDA_VERSION_ARG-runtime-ubuntu22.04
+ARG CUDA_VERSION_ARG
+ARG RAPIDS_VERSION_ARG
+  # Should be first 4 digits (e.g. 24.06)
+ARG NCCL_VERSION_ARG
```

Comment:
This comment is totally a minor-personal-preference thing, feel free to ignore it... but I feel that suffixing all of these with `*_ARG` is superfluous, and makes it a bit harder to read. Consider these side-by-side:

current:

```dockerfile
ARG CUDA_VERSION_ARG=notset
FROM nvidia/cuda:$CUDA_VERSION_ARG-runtime-ubuntu22.04
ARG CUDA_VERSION_ARG
ARG RAPIDS_VERSION_ARG
  # Should be first 4 digits (e.g. 24.06)
ARG NCCL_VERSION_ARG
```

proposed:

```dockerfile
ARG CUDA_VERSION=notset
FROM nvidia/cuda:$CUDA_VERSION_ARG-runtime-ubuntu22.04
ARG CUDA_VERSION
ARG RAPIDS_VERSION
  # Should be first 4 digits (e.g. 24.06)
ARG NCCL_VERSION
```

In my opinion, the second form is easier to read at a glance.

Reply:
I rename the args to remove the `_ARG` suffix.